README.rst

```diff
@@ -148,7 +148,35 @@ When your environment is set up, you can run the import process and start the se
    # Start the server
    flask run
 
-The service will now listen on ```http://localhost:5000``.
+The service will now listen on ``http://localhost:5000``.
+
+
+Windows with remote PostGIS set up
++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Rename ``./openelevationservice/server/ops_settings.sample.yml`` to ``ops_settings.yml``:
+
+- Set ``coord_precision`` to ``0.000833333333``.
```

Comment:
It would be interesting to explain this coord_precision also in the Linux section.

Reply:
DONE